recipe/build-lib.sh

```diff
@@ -1,5 +1,20 @@
 CUDA_CONFIG_ARG=""
 if [ ${cuda_compiler_version} != "None" ]; then
+    # for documentation see
+    # docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#building-for-maximum-compatibility
+    # docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#major-components__table-cuda-toolkit-driver-versions
+    # docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list
+
+    # default of `./configure --with-cuda-arch=...` corresponds to compute_{35,52,60,61,70,75}
+    ARCHES=(35 52 60 61 70)
```

Comment:
@teju85 
If I may ask another question: in the [list](https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list) of arches, I also see `sm_{50, 53, 62, 72}`. Should these be added as well?

Reply:
53, 62, 72 are for our Jetson product family. So, those should be used for the ARM builds (not sure if you are planning to package for aarch64 too)

50 is certainly missing from the above list. Good catch. I think it is safer and better to add it also to complete the support matrix.

Reply:
Thanks!

With "complete the matrix" do you mean all of them, or are 53/62/72 *only* necessary for aarch?

Aarch-builds are dependent on having `cudatoolkit` and `nvcc` packaged within conda-forge first, but will hopefully be added eventually.

Reply:
Ah.. my bad. I meant completing the support matrix for the x86 packages. 53/62/72 are only for aarch64 packages.

Reply:
OK, thanks! Pushed a fix & a comment in the build-script to that effect.